lib/content-services/search/components/search-filter/search-filter.component.ts

```diff
@@ -342,4 +356,55 @@ export class SearchFilterComponent implements OnInit, OnDestroy {
 
         return `${fieldName}:${startLimit}"${start}" TO "${end}"${endLimit}`;
     }
+
+    private getApiResponseField(context: ResultSetContext, itemType: string, fieldLabel: string): GenericFacetResponse {
+        return (context.facets || []).find((response) => response.type === itemType && response.label === fieldLabel) || {};
+    }
+
+    private getAlreadyDisplayedField(itemType: string, fieldLabel: string): FacetField {
+        return (this.responseFacets || []).find((response) => response.type === itemType && response.label === fieldLabel);
+    }
+
+    private updateExistingBuckets(responseField, responseBuckets, alreadyExistingField, alreadyExistingBuckets) {
+        const shouldDelete = [];
```

Comment:
Rename the variable to express intent better
```suggestion
        const bucketsToDelete = [];
```